test/profile.routes.test.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Dependencies.
+ */
+var app = require('../index');
+var expect = require('chai').expect;
+var request = require('supertest-as-promised');
+var utils = require('../test/utils.js')();
+
+/**
+ * Variables.
+ */
+var userData = utils.data('user1');
+var groupData = utils.data('group1');
+var models = app.set('models');
+
+/**
+ * Tests.
+ */
+describe('groups.routes.test.js', () => {
```

Comment:
should be `profile.routes.test.js`, so we don't misidentify the test when it fails.